docs/api/README.md

```diff
@@ -176,13 +176,20 @@ const store = new Vuex.Store({ ...options })
 
 -  `subscribe(handler: Function, options?: Object): Function`
 
-  Subscribe to store mutations. The `handler` is called after every mutation and receives the mutation descriptor and post-mutation state as arguments:
+  Subscribe to store mutations. The `handler` is called after every mutation and receives the mutation descriptor and post-mutation state as arguments.
+
+:::warning WARNING
```

Comment:
I think when you actually want to unsubscribe when unregistering the Vuex modules. So instead of making this "warning", how about we just add a description to the `unsubscribe` function?

---

Subscribe to store mutations. The `handler` is called after every mutation and receives the mutation descriptor and post-mutation state as arguments.

The `subscribe` method will return `unsubscribe` function, which can be used to stop the subscription.

Reply:
Thanks! I understand that Vuex is quite decoupled from Vue components, but I think mentioning the need to call unsubscribe when called from a component is still helpful. 

When I was debugging a leak caused by one of our components calling `this.$store.subscribeAction` on create, it wasn't obvious the subscription wasn't cleaned up after being destroyed. I pushed up a change that removes the warning block and extends your suggestion with:
> The `subscribe` method will return an `unsubscribe` function, which should be called when the subscription is no longer needed. For example, when unregistering a Vuex module or before destroying a Vue component.

I will make any additional changes as needed.

Reply:
Made new comment! Please have a look 🙏